### PR TITLE
Updated unfilled option error

### LINKED ISSUE #37: Incorrect error message rendering for unfilled options
### Description
`Case: 1`
If option1 is left empty and option2 is filled then error message should be on option1 input tag and not on option2 input tag.

`Case: 2`
If option1 and option2 are left empty and some other options are filled then error message should be on option1 input tag and not on option2 input tag.

### PR DESCRIPTION
**Pull request by ashwaniYDV**
Closes #37

### CODE REVIEW

**Comment by ashwaniYDV** [comment]
@sampaiodiego Please review

File: src/lib/createPollMessage.ts [hunk 1]
```diff
@@ -36,9 +36,16 @@ export async function createPollMessage(data: IUIKitViewSubmitIncomingInteractio
     }
 
     if (options.length === 1) {
-        throw {
-            'option-1': 'Please provide one more option',
-        };
+        if (!state.poll['option-0'] || (state.poll['option-0'] && state.poll['option-0'] === '')) {
```
Comment (sampaiodiego) [inline src/lib/createPollMessage.ts]: you don't need to test the existence of `state.poll['option-0']` two times.. so you could change to this
```suggestion
        if (!state.poll['option-0'] || state.poll['option-0'] === '') {
```

File: src/lib/createPollMessage.ts [hunk 2]
```diff
@@ -36,9 +36,16 @@ export async function createPollMessage(data: IUIKitViewSubmitIncomingInteractio
...
+            throw {
+                'option-0': 'Please provide one more option',
+            };
+        }
+        if (!state.poll['option-1'] || (state.poll['option-1'] && state.poll['option-1'] === '')) {
```
Comment (sampaiodiego) [inline src/lib/createPollMessage.ts]: same as above:
```suggestion
        if (!state.poll['option-1'] || state.poll['option-1'] === '') {
```

**Comment by ashwaniYDV** [comment]
hi @sampaiodiego I have done the requested changes :)

**Pull request merged**